crates/bevy_ui/src/focus.rs

```diff
@@ -35,8 +35,8 @@ use smallvec::SmallVec;
 #[derive(Component, Copy, Clone, Eq, PartialEq, Debug, Reflect, Serialize, Deserialize)]
 #[reflect(Component, Serialize, Deserialize, PartialEq)]
 pub enum Interaction {
-    /// The node has been clicked
-    Clicked,
+    /// The node has been pressed
```

Comment:
Nit: It could be good to also update this comment to really make it clear that this is not like a click (i.e. it does not capture a press-release action).

Reply:
Amended comment to hopefully be clearer

Reply:
Thanks! Just left a quick followup  comment on that addition